responsibleai/tests/test_model_analysis.py

```diff
@@ -78,7 +78,9 @@ def test_model_analysis_binary(self, manager_type):
                                manager_type, manager_args,
                                classes=classes)
 
-    @pytest.mark.parametrize('manager_type', [ManagerNames.COUNTERFACTUAL])
+    @pytest.mark.parametrize('manager_type', [ManagerNames.COUNTERFACTUAL,
+                                              ManagerNames.EXPLAINER,
+                                              ManagerNames.ERROR_ANALYSIS])
```

Comment:
I'm not sure how this test passes but it actually shouldn't even work yet.  I will be working on enabling this scenario for error analysis.

Reply:
I can remove the ERROR_ANALYSIS part. But if regression is not supported as yet, we should handle it in ErrorAnalysis Manager as an user error and give proper message to the user.

Reply:
I'm working on adding support right now as a mini-project, but it involves making changes to UI and SDK.  Since I will add it soon I think it's ok to not have an error for now.